### PR TITLE
fix: add vc status list for each profile

### PR DESCRIPTION
**Pull request by fqutishat**
Signed-off-by: Firas Qutishat <firas.qutishat@securekey.com>

### CODE REVIEW

**Review by sudeshrshetty — APPROVED** [review]

File: pkg/doc/vc/status/csl/csl.go [hunk 1]
```diff
@@ -23,7 +23,8 @@ import (
 )
 
 const (
-	vcContext = "https://www.w3.org/2018/credentials/v1"
+	vcContext               = "https://www.w3.org/2018/credentials/v1"
+	jsonWebSignature2020Ctx = "https://w3c-ccg.github.io/lds-jws2020/contexts/lds-jws2020-v1.json"
```
Comment (troyronda) [inline pkg/doc/vc/status/csl/csl.go:27]: this isn't the canonical context

it's: https://w3id.org/security/jws/v1

File: pkg/doc/vc/status/csl/csl.go [hunk 2]
```diff
@@ -296,6 +297,11 @@ func (c *CredentialStatusManager) createVC(vcID string,
 	profile *vcprofile.DataProfile) (*verifiable.Credential, error) {
 	credential := &verifiable.Credential{}
 	credential.Context = []string{vcContext, Context}
+
+	if profile.SignatureType == vccrypto.JSONWebSignature2020 {
```
Comment (troyronda) [inline pkg/doc/vc/status/csl/csl.go:301]: note: BBS would also need another context if that key is used.

**Review by troyronda — APPROVED** [review]

File: pkg/doc/vc/status/csl/csl.go (hunk 1 above)
Comment (fqutishat) [reply pkg/doc/vc/status/csl/csl.go]: done. change it to ttps://w3id.org/security/jws/v1

Comment (troyronda) [reply pkg/doc/vc/status/csl/csl.go:27]: Looks like it got reverted.

**Pull request merged**